lib/runners/processor.rb

```diff
@@ -94,30 +82,12 @@ def self.ci_config_section_name
       self.name
     end
 
-    def ci_config_path
-      new_config_path = working_dir + "sider.yml"
-      if new_config_path.file?
-        new_config_path
-      else
-        working_dir + "sideci.yml"
-      end
-    end
-
-    def ci_config_path_name
-      if ci_config_path.file?
-        File.basename(ci_config_path)
-      else
-        "sider.yml"
-      end
-    end
-
-    def ci_section(default = {})
-      section = ci_config&.dig("linter", self.class.ci_config_section_name)
-      default.deep_merge(Hash(section))
+    def ci_section
+      Hash(config.content.dig(:linter, self.class.ci_config_section_name.to_sym))
```

Comment:
[question] Is this equivalent? 👇 

```suggestion
      config.content.dig(:linter, self.class.ci_config_section_name.to_sym) || {}
```

Reply:
Yes, this is the same. Your suggestion is better than me because it's explicit that the fallback is `{}`.